packages/fxa-settings/src/components/DataBlock/index.stories.tsx

```diff
@@ -3,28 +3,42 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from 'react';
-import { storiesOf } from '@storybook/react';
```

Comment:
Irrelevant, but since I'm a fan of <kbd>grep</kbd>:
```sh
git grep -n "storiesOf" packages/**/*.stories.* | grep "import "

packages/fxa-payments-server/src/components/AppLayout/index.stories.tsx:2:import { storiesOf } from '@storybook/react';
packages/fxa-payments-server/src/components/DialogMessage/index.stories.tsx:6:import { storiesOf } from '@storybook/react';
packages/fxa-payments-server/src/routes/Product/IapRoadblock/index.stories.tsx:2:import { storiesOf } from '@storybook/react';
packages/fxa-payments-server/src/routes/Product/SubscriptionChangeRoadblock/index.stories.tsx:2:import { storiesOf } from '@storybook/react';
packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.stories.tsx:3:import { storiesOf } from '@storybook/react';
packages/fxa-payments-server/src/routes/Product/index.stories.tsx:2:import { storiesOf } from '@storybook/react';
packages/fxa-payments-server/src/routes/Subscriptions/index.stories.tsx:6:import { storiesOf } from '@storybook/react';
packages/fxa-react/components/AppErrorDialog/index.stories.tsx:2:import { storiesOf } from '@storybook/react';
packages/fxa-react/components/Footer/index.stories.tsx:6:import { storiesOf } from '@storybook/react';
packages/fxa-react/components/Head/index.stories.tsx:6:import { storiesOf } from '@storybook/react';
packages/fxa-react/components/Header/index.stories.tsx:2:import { storiesOf } from '@storybook/react';
packages/fxa-react/components/LinkExternal/index.stories.tsx:2:import { storiesOf } from '@storybook/react';
packages/fxa-react/components/LoadingSpinner/index.stories.tsx:2:import { storiesOf } from '@storybook/react';
packages/fxa-react/lib/hooks.stories.tsx:2:import { storiesOf } from '@storybook/react';
```

Reply:
Thanks for confirming that there are no more instances of `storiesOf` in fxa-settings 👍 I'd been inspired by your grepping and had used a rudimentary `grep` to build the initial list of components that needed to be updated 😉